packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -42,6 +43,28 @@ class PackagesCommand extends FlutterCommand {
 
   @override
   Future<FlutterCommandResult> runCommand() async => null;
+
+  @override
+  Future<Map<String, String>> get usageValues async {
+    final Map<String, String> usageValues = <String, String>{};
+    final String target = findProjectRoot(
+      argResults.rest.length == 1 ? argResults.rest[0] : null
```

Comment:
Create a self documenting local var for what this is supposed to represent semantically?

Reply:
Done